### DISCUSSION TITLE
Multiple Discord Webhooks

### DISCUSSION THREAD

**Discussion by bsumners** [lloesche/valheim-server-docker#287] [Q&A] [answered]
Is it possible to send an event message to multiple Discord webhooks?

Right now, I have it working to send to just one. I would like to send to a second.

**Comment by bsumners** [comment, marked as answer]
Nevermind, I figured it out just by adding that second webhook right after the first...

**Comment by Jmag103** [comment]
tried this and didn't work for me. Anyone have any idea?

**Comment by Jmag103** [comment]
Got it working, make required params, then list both webhooks at the end like so

```
PRE_BOOTSTRAP_HOOK=curl -sfSL -X POST -H "Content-Type: application/json" -d "{\"username\":\"hugin\",\"content\":\"$(eval echo $DISCORD_SERVER_STARTING_MSG)\"}" "$DISCORD_WEBHOOK" "$DISCORD_WEBHOOK2"
POST_SERVER_LISTENING_HOOK=curl -sfSL -X POST -H "Content-Type: application/json" -d "{\"username\":\"hugin\",\"content\":\"$(eval echo $DISCORD_SERVER_STARTED_MSG)\"}" "$DISCORD_WEBHOOK" "$DISCORD_WEBHOOK2"
```
